src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -161,13 +170,70 @@ static unsigned uct_cuda_copy_iface_progress(uct_iface_h tl_iface)
     unsigned max_events = iface->config.max_poll;
     unsigned count;
 
-    count = uct_cuda_copy_progress_event_queue(&iface->outstanding_d2h_cuda_event_q,
+    count = uct_cuda_copy_progress_event_queue(iface, UCT_CUDA_COPY_D2H_STREAM,
                                                max_events);
-    count += uct_cuda_copy_progress_event_queue(&iface->outstanding_h2d_cuda_event_q,
+    count += uct_cuda_copy_progress_event_queue(iface, UCT_CUDA_COPY_H2D_STREAM,
                                                 (max_events - count));
     return count;
 }
 
+static void uct_cuda_copy_common_cb(void *cuda_copy_iface)
```

Comment:
it seems that we don't need this wrapper, since it could be used directly in the `myHostCallback()`/`myHostFn()`